### DISCUSSION TITLE
Active tab color inconsistency in Dark Theme

### DISCUSSION THREAD

**Discussion by eakwarren** [musescore/MuseScore#23372] [Ideas]
Tab panels in the Light theme consistently show active tabs as a lighter color. Below, the Instruments and Analysis tabs are the active tabs.
<img width="590" alt="Screenshot 2024-06-26 at 5 18 51 PM" src="https://github.com/musescore/MuseScore/assets/1768096/7c6dcedf-59bb-44b2-99f2-9f7f18daff3c">

In the Dark theme, however, the tab colors are inconsistent. Same tabs are selected as above, but it appears the * The Lord Of the Rings - tab is active and not the Analysis tab.
<img width="590" alt="Screenshot 2024-06-26 at 5 19 20 PM" src="https://github.com/musescore/MuseScore/assets/1768096/4ed5371c-56a1-414a-98c0-3017f3287d40">

Bonus: The active file/part tab text should be bold to be consistent with the sidebar active tab.

**Comment by cbjeukendrup** [comment]
It would be better to log this as an issue / bug report, because that increases the chance of it being seen by the design team.

**Reply by eakwarren** [reply]
Thank you. Initially I didn't see Issues listed in the top menu, but diving into the [MuseScore repo](https://github.com/musescore/MuseScore) proper, I now see the Issues link. [I've added there.](https://github.com/musescore/MuseScore/issues/23377)
